### PR TITLE
Flag to allow using deprecated docker runtime

### PR DESCRIPTION
**Pull request by ffilippopoulos**

### CODE REVIEW

File: common.tf [hunk 1]
```diff
@@ -227,6 +235,7 @@ data "ignition_file" "sysctl_kernel_vars" {
 fs.inotify.max_user_watches=1048576
 fs.inotify.max_user_instances=8192
 vm.max_map_count=262144
+user.max_user_namespaces=0
```
Comment (george-angel) [inline common.tf:238]: Sorry about being picky - should this be a separate commit so its not reverted accidentally?

We should cherry-pick from the release-v1.21 branch?

Comment (ffilippopoulos) [reply common.tf:238]: yes, definitely. I will remove it and rebase to clean it from history. Then we can cherry pick. I was working in parallel with HH and threw this in here to avoid reverting (or waiting :))

Comment (ffilippopoulos) [reply common.tf]: done

**Review by hectorhuertas — APPROVED** [review]

**Review by george-angel — APPROVED** [review]
